extra/Documentation/eng/TechInfo.ChangeTracking.md

```diff
@@ -0,0 +1,118 @@
+This temporary file keeps track of the changes to `eng\TechInfo.txt`. Please review the initial change list.
+I will be updating `TechInfo.txt` and this file as I continue translating topics in `FarEng.hlf.m4`.
+
+
+Deleted on 2019-09-22
+=====================
+
+Please double check! I could not find any traces. Are they obsolete?
+--------------------------------------------------------------------
+```
+1.1.6.(TechInfo #40)  (User menu caption)
+1.1.8.(TechInfo #52)  (Saved???/HistoryCount)
+1.2.0.(TechInfo #2)   (TemplatePluginsPath)
+1.2.21.(TechInfo #60) (System/IgnoreErrorBadPathName)
+1.2.22.(TechInfo #62) (System/UsePrintManager)
+1.2.26.(TechInfo #76) (System/FolderDeepScan)
+1.3.2.(TechInfo #53)  (System/Executor/Type)    - Should it be removed from Executor.*.farconfig?
```

Comment:
Obsolete, removed in 935. Yes, should be removed from other places as well.

Reply:
Thanks. Cleaned up `Executor.*.farconfig` files.